docs/content/concepts/assets/external-assets.mdx

```diff
@@ -124,7 +123,7 @@ height={1654}
 </TabItem>
 </TabGroup>
 
-### Fully-managed assets with external asset dependencies
+### Dagster-native assets with external asset dependencies
```

Comment:
Thoughts on this terminology?

Reply:
i can't think of anything better atm. "Dagster-managed"? "Dagster-materialized"? Listing out the actual decorators "@asset, @multi_asset, and @graph_asset with external ..."?